src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,43 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer lada = new Manufacturer("Lada", "Ukraine");
+        Manufacturer deo = new Manufacturer("Deo", "Ukraine");
+        manufacturerService.create(lada);
+        manufacturerService.create(deo);
```

Comment:
here and elsewhere you should use constructions like this:
object = objectService.create(object)

this way you have a fully initialized object, otherwise it will miss an ID and you may run into mistakes

Reply:
we just need to  check in our tables it works or not

Reply:
our models are mutable, if we set id on dao layer, we are able to get it here